.travis.yml

```diff
@@ -0,0 +1,23 @@
+sudo: required
+dist: trusty
+addons:
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
+language: node_js
+node_js:
+  - 7
+
+env:
+  global:
+    - API_KEY=AIzaSyDNGPh2ERYJq9Ei1tzDSNG-nOyYAJVhpY4
```

Comment:
todo: encrypt the key using this: https://docs.travis-ci.com/user/environment-variables/